sgx/urts/src/enclave.rs

```diff
@@ -324,27 +316,7 @@ impl SgxEnclave {
 
     fn exit(&self) {}
 
-    fn init(&self) {
-        // Store enclave id -> pathbuf in the map, for backtrace functionality
-        #[cfg(feature = "backtrace")]
-        self.store_id_and_path();
-    }
-
-    #[cfg(feature = "backtrace")]
-    fn store_id_and_path(&self) {
-        let mut lk = crate::backtrace::ENCLAVE_PATH_MAP
-            .lock()
-            .expect("Could not lock enclave path map!");
-        let maybe_old_path = lk.insert(self.id, self.path.clone());
-        if let Some(old_path) = maybe_old_path {
-            panic!(
-                "The eid is already in the enclave path map! eid: {} old_path: {} new_path: {}",
-                self.id,
-                old_path.display(),
-                self.path.display()
-            );
-        }
-    }
+    fn init(&self) {}
```

Comment:
is there any point to keeping these functions around?

exit() was here before but they're private and only called in this module so it seems like simple deletions

Reply:
good point, will remove